1.7/overview/what-is-dcos.md

```diff
@@ -37,6 +37,6 @@ You can provide a virtual IP and port for clients to communicate with your conta
 
 ## Package management
 
-Applications tend to be composed of more than a single process that has no configuration. Microservices can be made up of hundreds of processes all with their own configurations. Taking something like this and deploying it across a datacenter is a time consuming, delicate process. It is easy to misunderstand the implications of a specific configuration setting in your environment or just be unable to get the application running.
+Applications tend to be composed of more than a single process, and often values are hard-coded inside the application, or perhaps in a single configuration file. In contrast, microservices can be made up of hundreds of processes, each with their own configurations. Taking a collection of interdependent microservices and deploying them across a datacenter is a time consuming, delicate process. It is easy to misunderstand the implications of a specific configuration setting in your environment, or worse you may be unable to get the application running at all.
 
-One way to make sense of all this complexity is to create a package of the common applications you would like to take advantage of in your datacenter. There is a multitude of options such as databases and message queues that you build on top of. By bundling the processes and configuration into a single package, you can get started fast and be sure you’re following best practices.
+One way to make sense of all this complexity is to create a package of the common applications you would like to take advantage of in your datacenter, to simplify deployment and life cyle management. When bundling common applications into deployment packages, you also have many choices of technology you can build on top of, such as databases or messaging services. By bundling the processes and configurations into a single package, you can get started fast and be sure you’re following best practices.
```

Comment:
This reads awkwardly, and has a typo `...to simplify deployment and life cyle management`. I think it can be left off entirely.